TFT/src/User/Menu/LoadUnload.h

```diff
@@ -7,6 +7,15 @@ extern "C" {
 
 void menuLoadUnload(void);
 
+typedef enum

```

Comment:
I usually provide declarations in .c  if they are not general declaration used also by other .c

Reply:
Fixed